roulette.py

```diff
@@ -1,92 +0,0 @@
-import random
-
-def rouletteNombre (betUser,money): #boucle pour répétition si user sort de l'intervalle [1,36]   
-    choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    while choiceUser <1 or choiceUser >36: 
-        choiceUser=eval(input("Sur quel numéro voulez-vous poser votre jeton de 1 à 36 :"))
-    result=random.randrange(0,36)
-    print("La bille est tombée sur le", result)
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    if choiceUser==result:
-            betUser=betUser*36
-            money=money+betUser
-            print("vous avez gagné",betUser,"€")
-            return money
-    else:
-            print("vous avez perdu", betUser, "€")
-            return money
-
-def rouletteStart(): #demande la mise
-    print("faites vos jeux !")
-    betUser=eval(input("Entrer la mise en € : "))
-    return betUser
-
-def roulettePair(betUser,money): #jeu pair/impair
-    PI=["Pair","impair"]
-    choiceUser=eval(input("Précisez votre choix : Pair=0, Impair=1 : "))
-    result=random.randrange(0,36)
-    print("la bille est tombée sur le :",result)
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    result=result%2
-    print("Le résultat est",PI[result])
-    if result==choiceUser:
-        betUser=betUser*2
-        money=money+betUser
-        print("vous avez gagné", betUser,"€")
-        return money
-    else:
-        print("Vous avez perdu", betUser,"€")
-        return money
-
-def rouletteCouleur(betUser,money): #Jeu couleur
-    couleur=["vert","rouge","noir","rouge","noir","rouge","noir","rouge","noir","rouge","noir","noir","rouge","noir","rouge","noir","rouge","noir","rouge","rouge","noir","rouge","noir","rouge","noir","rouge","noir","rouge","noir","noir","rouge","noir","rouge","noir","rouge","noir","rouge"]
-    choiceUser=input("Précisez votre choix rouge ou noir ? : ")
-    result=random.randrange(0,36)
-    print("La bille est tombée sur", result,"de couleur",couleur[result])
-    if result==0:
-        print("la bille est tombée sur le 0, vous avez perdu")
-        return money
-    if couleur[result]==choiceUser:
-        betUser=betUser*2
-        money=money+betUser
-        print("vous avez gagné",betUser,"€")
-        return money
-    else:
-        print("Vous avez perdu", betUser,"€")
-        return money
-
-def roulette(betUser,money): 
-    choiceGame=eval(input("Sur quels jeux voulez-vous placer votre mise ? Couleur=0 Pair/Impair=1 Nombres=2 :"))
-    if choiceGame==0:
-        return rouletteCouleur(betUser,money)
-    elif choiceGame==1:
-        return roulettePair(betUser,money)
-    elif choiceGame==2:
-        return rouletteNombre(betUser,money)
```

Comment:
This is what we called an `if forest` and it should be avoided once the number of cases is 3 or more with a single simple parameter being checked (in your case you have 3 cases and a simple parameter, choiceGame).
What we can do instead is this :

```
ROULETTE_CHOICES = {
    0: rouletteCouleur,
    1: roulettePair,
    2: rouletteNombre,
}
choice_game = int(input("blabla"))
return ROULETTE_CHOICES[choice_game](bet_user, money)
```